mw-webapp/src/component/button/Button.cy.tsx

```diff
@@ -0,0 +1,36 @@
+import {Button} from "src/component/button/Button";
+
+const BUTTON_VALUE = "Click button";
+
+describe("<Button />", () => {
+  it("renders", () => {
+    const STUB_FUNCTION = cy.stub();
+    cy.mount(
+      <Button
+        value={BUTTON_VALUE}
+        onClick={STUB_FUNCTION}
+      />);
+  });
+
+  it("uses custom text for the button", () => {
+    const STUB_FUNCTION = cy.stub();
+    cy.mount(
+      <Button
+        value={BUTTON_VALUE}
+        onClick={STUB_FUNCTION}
+      />);
+    cy.get("button").should("contains.text", "Click button");
+  });
+
+  it("clicking", () => {
+    const STUB_FUNCTION = cy.stub();
+    cy.mount(
+      <Button
+        value={BUTTON_VALUE}
+        onClick={STUB_FUNCTION}
+      />);
+    cy.get("button").click().then(() => {
```

Comment:
Maybe better to use data-* attributes for selecting elements?
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
Changed. Thanks